sdk/purview/azure-analytics-purview-sharing/swagger/README.md

```diff
@@ -45,5 +45,7 @@ directive:
       to: "ReceivedShares_GetAllAttachedReceivedShares",
       from: "ReceivedShares_ListDetached",
       to: "ReceivedShares_GetAllDetachedReceivedShares",
+      from: "ShareResources_List",
+      to: "ShareResources_GetAllShareResources"
```

Comment:
All operations that return a paged response should have `list` as the prefix in Java, Python and JS. Only .NET uses `Get` prefix for paged response.

Reply:
Thank you for the guidance.  I have updated our transformations to remove `GetAll` and to use `List` instead in all instances where `GetAll` was used.